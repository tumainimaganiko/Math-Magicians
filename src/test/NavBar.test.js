import renderer from 'react-test-renderer';
import Navbar from 'components/NavBar';
import { MemoryRouter } from 'react-router-dom';


it('Renders a Navigation Bar', () => {
    const tree = renderer.create(
        <MemoryRouter>
            <Navbar />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
})