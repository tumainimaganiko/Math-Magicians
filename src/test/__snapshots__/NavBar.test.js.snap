// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a Navigation Bar 1`] = `
Array [
  <nav
    className="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <ul>
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/quotes"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
      <li>
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
    </ul>
  </nav>,
  <footer
    style={
      Object {
        "bottom": "0px",
        "position": "fixed",
        "right": "10px",
      }
    }
  >
    Crafted with code and creativity by
     
    <b>
      Tumaini Maganiko
    </b>
     
    .
  </footer>,
]
`;
